### ISSUE TITLE
Compile xontribs

### ISSUE THREAD

**Issue by anki-code** [xonsh/xonsh#3953]
Hi! xonsh is a giant snail in small conch!

I found that xonsh compile scripts and save it to the `~/.local/share/xonsh/xonsh_script_cache` directory. But xonsh not compiles xsh xontribs. It will be great to compile them too to reduce the speed of start xonsh!

## For community
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by scopatz** [comment]
Yeah! I just noticed this too, @anki-code.  This would be great to have!

**Comment by AsafFisher** [comment]
It also does not compile imports from the xonshrc! or imports in general...
Maybe we should add an `include` command for xonsh?
```xonsh
include xonsh_aliases.xsh
```
